src/oscontainer-deprecated-legacy-format.py

```diff
@@ -207,28 +204,22 @@ def oscontainer_build(containers_storage, tmpdir, src, ref, image_name_and_tag,
         subprocess.call(buildah_base_argv + ['rm', bid], stdout=subprocess.DEVNULL)
 
     if push:
-        print("Pushing container")
+        print("Pushing container to oci-archive")
         podCmd = buildah_base_argv + ['push']
```

Comment:
I use `podman image save` for this over in https://github.com/coreos/coreos-assembler/blob/3d742f1195e1d6ad892538f54c184ea5b1cfa625/src/cmd-remote-build-container#L70-L71

Reply:
I am trying to leave as much code untouched as possible. Unless there is a reason not to use podman, this kind of simplifies not having to revisit the arguments passed. For example looking at podman save the `--format` has different values for similar things than buildah, If I am understanding their doc correctly.